recipes-kernel/linux/linux-imx-headers_6.12.bb

```
@@ -7,10 +7,11 @@ New headers are installed in ${includedir}/imx."
 LICENSE = "GPL-2.0-only"
 LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
-SRC_URI = "git://github.com/nxp-imx/linux-imx.git;protocol=https;branch=${SRCBRANCH}"
-SRCBRANCH = "lf-6.6.y"
-LOCALVERSION = "-6.6.52-2.2.0"
-SRCREV = "e0f9e2afd4cff3f02d71891244b4aa5899dfc786"
+SRC_URI = "${LINUX_IMX_SRC}"
+LINUX_IMX_SRC ?= "git://github.com/nxp-imx/linux-imx.git;protocol=https;branch=${SRCBRANCH}"
+SRCBRANCH = "lf-6.12.y"
+LOCALVERSION = "-lts-${SRCBRANCH}"
```

In the past we've put the release version here and ignored what was in `meta-imx`.
```suggestion
LOCALVERSION = "-6.12.3-1.0.0"
```

---

Fixed